Deployment/src/Modules/EventHub/main.tf

```
@@ -0,0 +1,61 @@
+resource "azurerm_eventhub_namespace" "eventhub_namespace" {
+  name                = "${var.name}-namespace"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  sku                 = "Standard"
+  capacity            = 1
+  tags                = var.tags
+}
```

This resource is missing tags

```suggestion
  tags                = var.tags
}
```